src/styles/blocks/header.scss

```diff
@@ -0,0 +1,76 @@
+.header {
+  background-color: $background-color-special-white;
```

Comment:
Ensure that the SCSS variable `$background-color-special-white` is defined in your stylesheets. If not, you need to define it or replace it with a valid color value.